### ISSUE TITLE
Deprecated geo_polygon field in geopoint

### ISSUE THREAD

**Issue by ebadyano** [elastic/rally-tracks#558]
when running geopoint `append-no-conflicts` challnege we get an deprecation warning:

```
Deprecated field [geo_polygon] used, replaced by [[geo_shape] query where polygons are defined in geojson or wkt]
```

**Comment by ebadyano** [comment]
@elastic/es-analytics-geo not sure if you want to replace this?

**Comment by iverase** [comment]
Yes, we should replace it with a geo_shape query.

**Comment by craigtaverner** [comment]
This was fixed by #602
